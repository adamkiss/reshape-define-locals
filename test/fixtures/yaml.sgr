define-locals(type='yaml').
	key: simple value
	array:
		- item 1
		- item 2
		- item 3
	object:
		sub: value
		deep:
			property: value
	dash-key: yolo
html
	body
		#expressions
			ul: each(loop='item in array'): li {{ item }}
		div(attr='{{ key }}') {{ object.deep.property }}