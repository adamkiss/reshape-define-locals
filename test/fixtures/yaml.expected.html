<html><body><div id="expressions"><ul><li>item 1</li><li>item 2</li><li>item 3</li></ul></div><div attr="simple value">value</div></body></html>